sound/soc/sof/intel/tgl.c

```diff
@@ -26,6 +26,8 @@ const struct snd_sof_dsp_ops sof_tgl_ops = {
 	.probe		= hda_dsp_probe,
```

Comment:
In commit, "Fixes:" is only used when pointing to commit hashes. To refer to a bug URL, please use "BugLink: " instead.

Reply:
@kv2019i since we are here and I know on i.MX there might be similar issues, do you know how the HIFI DSP is reset / power on / power off on Intel platforms?
Does the BIOS takes care of this. Or DSP by itself can initiate the rest/power off procedure?

Reply:
@dbaluta Actual power-on is triggered via PCI MMIO writes, see hda-dsp.c:hda_dsp_core_power_down() in kernel. On DSP side, there is a sequence to ramp down resources and got a loop executing waiti, see for example platform_pm_runtime_power_off().

Reply:
@kv2019i fixes is a github construct, it should be used in PR description to help autoclose related bugs if possible. But yes i agree not in the commit message.